.cirrus.yml

```diff
@@ -1,3 +1,19 @@
+environment:
+  X8664_UBUNTU1804_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-x86-64-unknown-linux-ubuntu18.04-builder:20210714"
+  X8664_UBUNTU2004_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-x86-64-unknown-linux-ubuntu20.04-builder:20200830"
+  X8664_UBUNTU2204_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-x86-64-unknown-linux-ubuntu22.04-builder:20220426"
+  X8664_MUSL_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-x86-64-unknown-linux-musl-builder:20220611"
+  AARCH64_UBUNTU_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-aarch64-unknown-linux-ubuntu22.04-builder:20220428"
+  X8664_WINDOWS_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-x86-64-pc-windows-msvc-builder:20210430"
+  X8664_CROSS_ARM_UBUNTU_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-cross-arm:20210430"
+  X8664_CROSS_ARMHF_UBUNTU_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-cross-armhf:20210430"
+  X8664_CROSS_RISCV_UBUNTU_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-cross-riscv64:20220716"
+  X8664_ROCKY_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-x86-64-unknown-linux-rocky8-builder:20210707"
+  X8664_UNKNOWN_BUILDER_IMAGE_NAME: "dipinhora/ponyc-ci-x86-64-unknown-linux-gnu-builder:20200423"
+  X8664_FREEBSD13_IMAGE_NAME: "freebsd-13-0-release-amd64"
+  X8664_MACOS_MONTEREY_XCODE132_IMAGE_NAME: "monterey-xcode-13.2"
+  ARM64_MACOS_MONTEREY_XCODE1331_IMAGE_NAME: "ghcr.io/cirruslabs/macos-monterey-xcode:13.3.1"
```

Comment:
NOTE: CI will fail for most of the targets until these docker images are updated to point to the new ones that include `gdb`.

Reply:
done